### PR TITLE
Unpack to install dir

### PR DESCRIPTION
**Pull request by jhixson74**
PR https://github.com/openshift/installer/pull/5715 with fixes for terraform path. 

https://issues.redhat.com/browse/CORS-1952

### CODE REVIEW

File: pkg/destroy/bootstrap/bootstrap.go [hunk 1]
```diff
@@ -64,6 +70,7 @@ func Destroy(dir string) (err error) {
 		tempDir, err := ioutil.TempDir("", fmt.Sprintf("openshift-install-%s-", stage.Name()))
 		if err != nil {
 			return errors.Wrap(err, "failed to create temporary directory for Terraform execution")
+
```
Comment (patrickdillon) [inline pkg/destroy/bootstrap/bootstrap.go:73]: extra line

**Review by patrickdillon — COMMENTED** [review]
/approve

This looks good. Can you just remove the extra line that snuck in?

**Comment by patrickdillon** [comment]
Oops. I hit CTRL + Enter instead of enter with my earlier comment.

This looks good. I tested with symlinks, creating a `c` dir with the install assets and then a symlink `s/c2` dir and installed with `openshift-install create cluster --dir s/c2` with no problems.

We should merge this as a single commit. I'm looking into now whether the tide merge label can automatically squash all commits, or whether those commits need to be labeled as fixups.

File: pkg/destroy/bootstrap/bootstrap.go (hunk 1 above)
Comment (jhixson74) [reply pkg/destroy/bootstrap/bootstrap.go]: Fixed.

**Comment by patrickdillon** [comment]
@jhixson74 I think `/label tide/merge-method-squash` will only squash fixup commits. Can you either squash the commit manually or make it a fixup? Thanks!

**Comment by janosdebugs** [comment]
FYI @mburman5

**Pull request merged**